WordPress/src/main/java/org/wordpress/android/util/WPUrlUtils.java

```diff
@@ -0,0 +1,45 @@
+package org.wordpress.android.util;
+
+import java.net.URI;
+import java.net.URL;
+
+public class WPUrlUtils {
```

Comment:
When you set a static method synchronized, it's the same as synchronizing over the class object - e.g. the next method is equivalent to:
```java
public static boolean safeToAddWordPressComAuthToken(String url) {
    synchronized (WPUrlUtils.class) {
        return UrlUtils.isHttps(url) && isWordPressCom(url);
    }
}
```

So in that case, because there is only "utility" methods (and no field in the class object), it's not necessary.